android/src/main/java/app/edge/reactnative/MyMoneroPackage.java

```diff
@@ -0,0 +1,21 @@
+package app.edge.reactnative;
```

Comment:
Shouldn't this package name be `app.edge.reactnativemonerocore`? To prevent conflict with other possible native libraries that we publish

Reply:
I figured the class name would be the unique thing, but I can easliy change this.